pyproject.toml

```diff
@@ -1,3 +1,58 @@
+[build-system]
+requires = ["setuptools<69.3", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lumicks.pylake"
+version = "1.5.1"
+requires-python = ">= 3.10"
+authors = [
+    {name = "Lumicks B.V."},
+    {email = "devteam@lumicks.com"},
+]
+readme = {file="README.md", content-type = "text/markdown"}
+license = {file = "license.md"}
+keywords = ["optical tweezers", "kymographs", "data analysis", "lumicks"]
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "pytest>=3.5",
+    "h5py>=3.4, <4",
+    "numpy>=1.24, <2",  # 1.24 is needed for dtype in vstack/hstack (Dec 18th, 2022)
```

Comment:
Not for this PR, but it reminded me: think about lifting this upper limit soon. `numpy` v2.1.0 is already out. In general, v2.x brings some really nice performance improvements for Mx Macs and with a much smaller package size (thanks to using the Accelerate SDK).

Reply:
Yeah, I will. I'll play around with it soon, but indeed, I don't want to piggyback that now.